OIPA/iati/models.py

```diff
@@ -53,6 +53,7 @@ class Activity(models.Model):
     hierarchy = models.SmallIntegerField(choices=hierarchy_choices, default=1, null=True)
     # last_updated_datetime = models.CharField(max_length=100, default="")
     # value_date = models.DateField(null=True)
+    last_updated = models.DateTimeField(null=True, auto_now=True)
```

Comment:
I don't mind it being added to the model if necessary, but with the current name it will be confusing with both the last_updated and last_updated_datetime in there. Even for me I still don't know what this date represents atm; Is it the last time the search index was updated to include this activity? Or the last time the parser updated this activity?